packages/flutter_reactive_ble/pubspec.yaml

```diff
@@ -21,8 +21,8 @@ dependencies:
     sdk: flutter
   functional_data: ^1.0.0
   meta: ^1.3.0
-  reactive_ble_mobile: ^5.2.0
-  reactive_ble_platform_interface: ^5.2.0
+  reactive_ble_mobile: ^5.3.0
+  reactive_ble_platform_interface: ^5.3.0
```

Comment:
please also update example/pubspec.yaml app

Reply:
Done, but it doesn't make much sense to delay merging and releasing only because of this, considering that example app has `dependency_overrides` pointing to local version of packages 😄

Reply:
Let's do it right, also can you please run `./bin/quality_checks.sh`

Reply:
`pubspec.lock` did not change

Reply:
Indeed , Thanks lot!